Client/Assets/Search/scrape_plugins.py

```diff
@@ -14,7 +14,18 @@
 import subprocess
 import urllib
 
-header = """\
+# Paths for en-US plugins included in the core Android repo.
+EN_PLUGINS_DIR_URL = "https://hg.mozilla.org/releases/mozilla-aurora/file/tip/mobile/locales/en-US/searchplugins"
```

Comment:
tip is generally safe on mozilla-aurora, but for consistency you should use 'default' (don't try that on beta though, tip and default are two different beasts)